data/maps/CeladonCity_Gym/text.inc

```diff
@@ -19,37 +19,37 @@ CeladonCity_Gym_Text_ErikaDefeat::
     .string "I concede defeat.\l"
     .string "You are remarkably strong.\p"
     .string "I must confer on you the\n"
-    .string "RAINBOWBADGE.$"
```

Comment:
Do we want to leave this as 1 word, or break it up into Rainbow Badge? The item name is never seen anywhere, and in reality they are two words